java/ql/src/semmle/code/java/frameworks/YamlBeans.qll

```diff
@@ -0,0 +1,20 @@
+/**
+ * Provides classes and predicates for working with the YamlBeans framework.
+ */
+
+import java
+
+/**
+ * The class `com.esotericsoftware.yamlbeans.YamlReader`.
+ */
+class YamlReader extends RefType {
```

Comment:
Perhaps rename it to `YamlBeansReader` to make it clear that this class belongs to this library when reading `UnsafeDeserialization` files